docs/shared/revisions.txt

```diff
@@ -26,6 +26,7 @@
 [cols="2",options="header"]
 |===
 | Version    | Date
+| 2.1.0      | TBD
 | 2.0.1      | July 7, 2016
 | 2.0.0      | June 6, 2016
 | 1.3.0      | January, 2016  
```

Comment:
I guess my preference was to remove this section, so we don't try to track release dates in documents.

Though it looks like this file is shared by all the docs?  That is less bad than I thought.  We could keep it if updating it is added to https://cwiki.apache.org/confluence/display/TRAFODION/Create+Release#CreateRelease-UpdateProjectWebSite